### DISCUSSION TITLE
八字、星座与MBTI - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by yufree** [yufree/yufree.cn#348] [General]
八字、星座与MBTI - Miao Yu | 于淼 
 https://yufree.cn/cn/2020/02/12/intj/

**Comment by yufree** [comment]
`Originally posted by yihui on 2020-02-12 15:48:05
        `
我一直记不住 MBTI 这个测试本身的名字，但我对 INTJ
印象却很深刻。不知道是因为它也差不多是我的类型（所以我通常关注同样类型的人，他们说过
INTJ 然后我记住了），还是因为它看起来像声明一个整型变量 int j，哈哈。

**Reply by yufree** [reply]
`Originally posted on 2020-02-12 15:56:43
        `
哈哈，我也记不住MBTI这名字，写的时候现搜INTJ才搜到的，但确实没想到整型变量这个梗。

**Comment by yufree** [comment]
`Originally posted by Liechi Zhang on 2020-02-13 04:11:00
        `
毕加索为格特鲁德·斯坦因画了幅肖像。看完画，斯坦因说：&quot;这画不像我啊。&quot;毕加索说：&quot;夫人，别担心，您会越来越像这幅画。&quot;

我觉得这类&quot;画&quot;一般都不像，我也不想长得跟&quot;画&quot;一样，所以过了初中就不玩这种测性格的游戏了。

**Reply by yufree** [reply]
`Originally posted on 2020-02-17 16:25:29
        `
性格差异确实会对交流产生影响，而且很多人对这玩意的热衷到了我不能理解的程度，很多人都需要一个哪怕虚幻的解释来接纳现状。

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2020-02-18 05:10:46
        `
可能人心里都希望有个简单的框架可以解释世界和人，伴随可解释而来的是可预测，随着可预测而来的是安全感，即使是虚幻的。比如我的以上说法就是企图用一个简单的模型来解释那些热衷性格测试的人的行为，我不知道以上说法对不对，但既然我考虑了一下这个问题，我会期待自己多少能理解一些东西。

我觉得知识背景对交流的影响可能大于性格，有时候彼此默认对方知道的东西和关心的重点跟实际会有极大的差别。

**Reply by yufree** [reply]
`Originally posted on 2020-02-18 21:11:10
        `
确实，我也感觉知识背景会对交流产生影响，但大概也会影响性格。对一件事了解越多，性格上可能更激进或更平和，这里面存在一个反馈加强或减弱，最近我就发现很多之前自以为很了解的人因为疫情的事性格变化很大，或者说我本来就不了解他们。

**Reply by yufree** [reply]
`Originally posted by Qins on 2020-02-24 02:36:41
        `
&gt; 对一件事了解越多，性格上可能更激进或更平和，这里面存在一个反馈加强或减弱

程序员的性格和思维，很可能有受程序逻辑的影响

**Comment by yufree** [comment]
`Originally posted by Qins on 2020-02-17 13:45:02
        `
INTP，怎么觉得INT这么多😂

**Reply by yufree** [reply]
`Originally posted on 2020-02-17 16:27:47
        `
具体到独立博客作者与读者，N不敢说，IT比例应该比较高。

**Comment by hi-jingxian** [comment]
我的mbti和八字好矛盾，遂google，居然搜到这篇文章，哈哈哈。人好像有时候是会为自己的行为寻求一个标签，大概这样能获取安全感。不过用标签定义一个人就太片面，所以不喜欢那些问了mbti之后就“噢，那你一定很xxx”之类的对话。

**Reply by yufree** [reply]
额，把mtbi写成mbti是我四年前的笔误，看回复似乎这个笔误不唯一。

**Reply by hi-jingxian** [reply]
确实是mbti啦你没写错

**Reply by yufree** [reply]
嗯，我搞混了，mtbi是脑震荡（mild traumatic brain injury）的缩写。

**Comment by wenping-zou** [comment]
同上一楼哈哈~~我也是用 八字 chatgpt & astrology chatgpt 和 mbti 交叉验证，不同的是，算是对上了？仍google，“居然搜到这篇文章”，楼主要不要一试？也许会垂直入坑hhh
